server/api/post.ts

```diff
@@ -0,0 +1,40 @@
+import { db } from "../database";
+import { isPost } from "../../server/types/DatabaseTypes";
+import { Request, Response } from "express";
+
+let firstPost: boolean = true;
+let postID: number;
+
+/**
+ * Creates a post, given valid Post information.
+ * @param {Request} req Requires a valid Post in the JSON body.
+ * @param {Response} res Responds with the post ID in the text field if successful, otherwise returns status code 500.
+ */
+function createPost(req: Request, res: Response) {
+    if(isPost(req.body)) {
+        db.exec(`INSERT INTO Posts VALUES ('${postID}', '${req.body.Username}', '${req.body.Content}', '${req.body.Picture}', '${req.body.Timestamp}')`);
+        console.log('[API] Post ' + postID + ' created');
+        res.status(200).send("" + postID);
+        ++postID;
```

Comment:
Prefer [db.run with params](https://github.com/TryGhost/node-sqlite3/wiki/API#runsql--param---callback) instead of exec with template strings. This can lead to SQL injections.

The `console.log` statement will happen immediately after the `db.exec`, since it's async. Place subsequent code in the callback of the `db.exec` statement. Not to mention, this will allow for error handling in the case of a Post not being able to be created.